### DISCUSSION TITLE
JSON for docker

### DISCUSSION THREAD

**Discussion by eXPerience83** [searxng/searxng#3996] [Q&A] [answered]
Hi!

I'm trying to use SEARXNG docker in Truenas Scale.
I need to add
FORMAT
 - JSON

to settings.ym or ar last using some environment variable.

This is to use web search on Open WebUI.

Is there any way to do it? I tried to modify on shell the file, but it's read-only.

**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by gamersalpha** [comment]
Hello, same question, how activate the json : 

![image](https://github.com/user-attachments/assets/c0460644-3a6c-4443-a960-4e33ca9058be)


but o nmy docker instance i don"t have it ?

![image](https://github.com/user-attachments/assets/c968f151-d5d7-48e7-82a5-af02cd196ba7)


can u know how to active it, in docker installation 

thanks

**Comment by allendema** [comment, marked as answer]
Answers here: https://github.com/searxng/searxng/discussions/3542#discussioncomment-11205515 / https://github.com/searxng/searxng/discussions/1789#discussioncomment-3688110
